packages/govuk-frontend/src/govuk/all.mjs

```diff
@@ -1,5 +1,4 @@
 import { version } from './common/govuk-frontend-version.mjs'
-import { nodeListForEach } from './common/index.mjs'
```

Comment:
![Animated GIF of a young Asian girl saying "The evil is defeated."](https://media.tenor.com/75EnGeBUdpgAAAAd/evil-defeated.gif)